amivapi/groups/model.py

```diff
@@ -47,15 +47,15 @@ class GroupMembershipAuth(AmivTokenAuth):
     """Auth for group memberships."""
 
     def has_resource_write_permission(self, user_id):
-        """All user can signup for groups.
+        """All user can enroll into groups.
```

Comment:
enroll stands with for, in or on, not with into
https://dictionary.cambridge.org/dictionary/english/enrol?a=british